calico-cloud/reference/architecture/design/l3-interconnect-fabric.mdx

```diff
@@ -435,7 +435,7 @@ routing protocols, or the like), and that routers in different ASs would
 not know how to reach one another unless they were directly connected.
 
 Based on that design point, routers in an iBGP peering relationship
-assume that they do not transit traffic for other iBGP routers in a
+assume that they do not transit traffic for other iBGProuters in a
```

Comment:
@samkookji77 I'm afraid that your fix for the latest problem introduced a new error. There are places in the docs that need to be "iBGProuters" and others that need to be "iBGP routers". Your fix changed all instances to "iBGProuters".

I suggest you start over. In your branch, do a hard reset to the commit before your HEAD commit. Then do your global replacement (BPG --> BGP). Commit again, force push to origin. 

Lots of moving parts with even the smallest changes! Thanks for sticking with it.